packages/databricks-vscode/src/test/e2e/auth.e2e.ts

```diff
@@ -0,0 +1,133 @@
+        );
+        await dismissNotifications();
+    });
+
+    it("should wait for a welcome screen", async () => {
+        const section = await getViewSection("CONFIGURATION");
+        const welcomeButtons = await browser.waitUntil(async () => {
+            const welcome = await section!.findWelcomeContent();
+            const buttons = await welcome!.getButtons();
+            if (buttons?.length >= 2) {
+                return buttons;
+            }
+        });
+        assert(welcomeButtons, "Welcome buttons don't exist");
+        const initTitle = await welcomeButtons[0].getTitle();
+        const quickStartTitle = await welcomeButtons[1].getTitle();
+        assert(
+            initTitle.toLowerCase().includes("initialize"),
+            "'initialize` button doesn't exist"
+        );
+        assert(
+            quickStartTitle.toLowerCase().includes("quickstart"),
+            "'quickstart' button doesn't exist"
+        );
+    });
+
+    it("should automatically login after detecting bundle configuration", async () => {
+        assert(process.env.DATABRICKS_HOST, "DATABRICKS_HOST doesn't exist");
+        await fs.writeFile(
+            bundleConfig,
+            BUNDLE.replace("_HOST_", process.env.DATABRICKS_HOST)
+        );
+        await waitForLogin("DEFAULT");
+    });
+
+    it("should create new profile", async () => {
+        // We create a new profile programmatically to avoid leaking tokens through screenshots or video reporters.
+        // We still trigger similar code path to the UI flow.
+        await browser.executeWorkbench(async (vscode) => {
+            await vscode.commands.executeCommand(
+                "databricks.connection.saveNewProfile",
+                "NEW_PROFILE"
+            );
+        });
+    });
+
+    it("should change profiles", async () => {
+        const section = (await getViewSection(
+            "CONFIGURATION"
+        )) as CustomTreeSection;
+        assert(section, "CONFIGURATION section doesn't exist");
+        const signInButton = await browser.waitUntil(
+            async () => {
+                const items = await section.getVisibleItems();
+                for (const item of items) {
+                    const label = await item.getLabel();
+                    if (label.toLowerCase().includes("auth type")) {
+                        return item.getActionButton("Sign in");
+                    }
+                }
+            },
+            {timeout: 10_000}
+        );
+        assert(signInButton, "Sign In button doesn't exist");
+        (await signInButton.elem).click();
+
+        const authMethodInput = await waitForInput();
+        const newProfilePick =
+            await authMethodInput.findQuickPick("NEW_PROFILE");
+        assert(
+            newProfilePick,
+            "NEW_PROFILE is absent in the quick pick selection"
+        );
+        await newProfilePick.select();
+        await waitForLogin("NEW_PROFILE");
+    });
```

Comment:
Can we add 1 more test to reload the window and see if the NEW_PROFILE is picked up this time.